gap/attr.gi

```diff
@@ -1582,3 +1582,125 @@ function(gr)
   od;
   return fail;
 end);
+
+InstallMethod(VertexConnectivity,
+"for a digraph",
+[IsDigraph],
+function(digraph)
+  local edges, edmondskarp, k, kappa, kappas, mindegv, Nv, outdegs, relabel,
+  sdigraph, set1v, x, y;
+
+  if DigraphNrVertices(digraph) = 0 then
+    return 0;
+  fi;
+
+  sdigraph := DigraphSymmetricClosure(DigraphRemoveAllMultipleEdges(digraph));
+  edges := DigraphEdges(sdigraph);
```

Comment:
It is probably better to avoid using `DigraphEdges` wherever possible, and instead use `DigraphOutNeighbours` (replacing any test `edge in DigraphEdges(sdigraph)` by `edge[1] in DigraphOutNeighbours(sdigraph)[edge[2]] or edge[2] in DigraphOutNeighbours(sdigraph)[edge[1]]`), or if you know you want to do lots and lots of checks if something is an edge in `sdigraph`, then you should so use `BooleanAdjacencyMatrix`.

Reply:
The time complexity of:

* `edge in DigraphEdges(sdigraph)` is big-O of the number of edges in `sdigraph`, 

* `edge[1] in DigraphOutNeighbours(sdigraph)[edge[2]] or edge[2] in DigraphOutNeighbours(sdigraph)[edge[1]]` is big-O of `Size(DigraphOutNeighbours(sdigraph)[edge[1]]) + Size(DigraphOutNeighbours(sdigraph)[edge[2]])` (or maybe log of this depending on `sdigraph`)

* `BooleanAdjacency(sdigraph)[edge[1]][edge[2]] or BooleanAdjacency(sdigraph)[edge[2]][edge[1]]` is constant.